LLM/OSS/Open_AI_OSS.py

```
@@ -347,6 +398,38 @@ def _phone_near_alias(text: str, aliases: set[str]) -> Optional[str]:
         return best
     return "-".join(phones[0].groups())
 
+def _hint_matches_line(label: str, body: str, hint: dict | None) -> bool:
+    if not hint:
+        return False
+    aliases = set(hint.get("aliases") or [])
+    if not aliases:
+        return False
+    return any((a in (label or "")) or (a in (body or "")) for a in aliases)
+
+def _fallback_phone_from_sub_answer(user_text: str, sub_answer: str, hint: dict | None = None) -> Optional[str]:
```

Ruff (0.15.1)

[warning] Unused function argument: `user_text`

(ARG001)